docs/explore/certificates.md

```diff
@@ -4,38 +4,12 @@ title: Certificates
 
 ## SSL Certificates
 
-Kubefirst takes advantage of a variety of security layers to make sure the platform is safe and secure for our users. One important and significant layer is the SSL certificates Kubefirst leverages to protect HTTPS requests. [Let's Encrypt](https://letsencrypt.org) is the certificate authority used to manage the certificates used for our ingressed services.
+In cloud versions of the kubefirst platform, we use [Cert Manager](https://cert-manager.io/) with a [Let's Encrypt](https://letsencrypt.org) clusterissuer for TLS encryption to all of our ingressed services. These certificates are requested automatically, and will auto-renew.
```

Comment:
Cert Manager = cert-manager
clusterissuer = ClusterIssuer

I would change "all of our ingressed services" to "all our services on Ingress". I'm fined using a verb that doesn't exist, but that people still used. In that case, just update add a capital I to ingressed.